code/modules/admin/verbs/infiltratorteam_syndicate.dm

```diff
@@ -0,0 +1,199 @@
+		new_syndicate_infiltrator.faction += "syndicate"
+		data_core.manifest_inject(new_syndicate_infiltrator)
+		if (syndicate_leader_selected)
+			var/obj/effect/landmark/warpto = pick(sit_spawns_leader)
+			new_syndicate_infiltrator.loc = warpto.loc
+			sit_spawns_leader -= warpto
+			team_leader = new_syndicate_infiltrator
+			to_chat(new_syndicate_infiltrator, "<span class='danger'>As team leader, it is up to you to organize your team! Give the job to someone else if you can't handle it. Only your ID opens the exit door.</span>")
+		else
+			to_chat(new_syndicate_infiltrator, "<span class='danger'>Your team leader is: [team_leader]. They are in charge!</span>")
+			teamsize--
+		to_chat(new_syndicate_infiltrator, "<span class='notice'>You have more helpful information stored in your Notes.</span>")
+		new_syndicate_infiltrator.mind.store_memory("<B>Mission:</B> [input] ")
+		new_syndicate_infiltrator.mind.store_memory("<B>Team Leader:</B> [team_leader] ")
+		new_syndicate_infiltrator.mind.store_memory("<B>Starting Equipment:</B> <BR>- Chameleon Jumpsuit ((right click to Change Color))<BR> - Agent ID card ((disguise as another job))<BR> - Uplink Implant ((top left of screen)) <BR> - Dust Implant ((destroys your body on death)) <BR> - Combat Gloves ((insulated, disguised as black gloves)) <BR> - Anything bought with your uplink implant")
+		var/datum/atom_hud/antag/sithud = huds[ANTAG_HUD_SIT]
+		sithud.join_hud(new_syndicate_infiltrator.mind.current)
+		ticker.mode.set_antag_hud(new_syndicate_infiltrator.mind.current, "hudsit")
+		new_syndicate_infiltrator.regenerate_icons()
+		num_spawned++
+		if (!teamsize)
+			break
+	if(spawn_sit_officer)
+		for(var/obj/effect/landmark/L in sit_spawns_officer)
+			var/mob/living/carbon/human/officer = create_syndicate_infiltrator(L, 1, 100, 1)
+			officer.key = src.key
+			officer.internal = officer.s_store
+			officer.internals.icon_state = "internal1"
+			officer.faction += "syndicate"
+			officer.equip_to_slot_or_del(new /obj/item/clothing/glasses/thermal(src), slot_glasses)
+			officer.equip_to_slot_or_del(new /obj/item/clothing/suit/space/rig/syndi/elite, slot_wear_suit)
+			officer.equip_to_slot_or_del(new /obj/item/clothing/head/helmet/space/rig/syndi/elite, slot_head)
+			officer.equip_to_slot_or_del(new /obj/item/clothing/mask/gas/syndicate, slot_wear_mask)
+			var/datum/atom_hud/antag/sithud = huds[ANTAG_HUD_SIT]
+			sithud.join_hud(officer.mind.current)
+			ticker.mode.set_antag_hud(officer.mind.current, "hudsit")
+			officer.mind.special_role = "Syndicate Officer"
+			officer.regenerate_icons()
+			to_chat(officer, "<span class='userdanger'>You have spawned as a Syndicate Officer. You can RP with the team, and brief them on the overall situation, but you should let the team leader organize things. Yours is a RP role, and you should NOT go with them.</span>")
```

Comment:
Who is them? Also the RP stuff kind of seems redundant, since an admin is going to be setting things up anyways.